src/test/integration/java/com/commercetools/payment/BasePaymentIT.java

```diff
@@ -98,6 +108,11 @@ public void tearDownAfterAll() {
 
     public void setUp() {
         initTenantConfigs();
+        cleanupProductsProductTypesTaxCategories(sphereClient);
```

Comment:
Curious why it was not necessary before?

Reply:
Probably because when running the tests, it was pointed towards the paypal_plus staging CT project which already had the expected product types. Therefore it was not required to create new product types in CT project. Therefore no need of clearing the old resources in CT project.